invenio_app_ils/circulation/search.py

```diff
@@ -74,6 +74,36 @@ def get_overdue_loans_by_doc_pid(self, document_pid):
             ),
         ).filter('range', end_date=dict(lt='now/d'))
 
+    def get_most_loaned_documents(self, from_date, to_date, bucket_size):
+        """Return aggregation of document_pids with most loans."""
+        search = self
```

Comment:
might be better to do something like
```
search = current_circulation.loan_search
```
because it is taking the class from records rest. All the method above are quite static too...
WDYT?